grpc/cpp-diatheke/CobaltGRPC.cmake

```diff
@@ -17,6 +17,12 @@ if(USE_SYSTEM_GRPC)
     add_executable(protoc ALIAS protobuf::protoc)
     add_executable(grpc_cpp_plugin ALIAS grpc::grpc_cpp_plugin)
 else()
+    # Make sure we build gRPC as a static library, regardless of
+    # whether Diatheke is built as a shared library.
+    cmake_policy(SET CMP0077 NEW)
```

Comment:
I was going to suggest adding a comment to briefly describe why setting this policy is necessary, but reading the cmake documentation about the policy made my eyes cross, so I think leave well enough alone. ;-)